lib/torque/provider/windshaft.js

```diff
@@ -295,14 +295,17 @@
                 .replace('{s}', subdomains[index])
 
       var extra = this._extraParams();
-      torque.net.get( url + (extra ? "?" + extra: ''), function (data) {
+      torque.net.get( url + (extra ? "?" + extra: ''), function (response) {
         prof_fetch_time.end();
-        if (data && data.responseText) {
-          var rows = JSON.parse(data.responseText);
-          callback(self.proccessTile(rows, coord, zoom));
+        if (response && response.responseText) {
+          var body = JSON.parse(response.responseText);
+
+          response.status === 429
+            ? errorCallback(body.errors_with_context[0])
```

Comment:
good for now, we'd probably want to send the tile info to draw the error tile, to be done on a separate PR